pallets/Polkadex_Ido/src/test.rs

```diff
@@ -271,51 +275,171 @@ fn test_claim_tokens() {
 
 #[test]
 fn test_show_interest_in_round() {
-    let balance: Balance = 100;
-    let investor_address: u64 = 4;
-    let block_num = 3;
+    // Amount to Raise for funding round
```

Comment:
Overview of test case in comment, no need to duplicate every assignment with an identical comment - give the big picture in a few lines